src/framework/button/_button.scss

```diff
@@ -12,6 +12,22 @@ $ctaButtonColor: #35d0a0;
     height: $iconSize;
     width: $iconSize;
     font-size: 10px;
+
+    &.icon-deal {
+      margin-right: 4px;
+      transform: scale(2.7);
+      backface-visibility: hidden;
```

Comment:
because backface-visibility simplifies the "over animation" to the front of that object. If you remove backfase-visibility the icon will be jumping everytime you over the other button due to the pseudo-element that is created when you over and the transform scale.